ilastik/workflows/objectClassification/objectClassificationWorkflow.py

```diff
@@ -430,7 +430,7 @@ def _inputReady(self):
                     continue
                 if not input_lane_slot[role].ready():
                     return False
-        return True and len(image_group_slot)
+        return True and len(image_group_slot) > 0
```

Comment:
```suggestion
        return bool(len(image_group_slot))
```

Or at least just `len(image_group_slot) > 0`...

Reply:
Oops ^^
`return True` is just leftover from previous versions where `return`ing `True` was all it did =)

Reply:
Maybe even go for `bool(image_group_slot)`? If there is no `__bool__` dunder, then the `__len__` is used anyway.